### ISSUE TITLE
Regression for multiple labels?

### ISSUE THREAD

**Issue by rnditdev** [charles9n/bert-sklearn#3]
Hi!
Is it possible to train a regression for multiple output variables? If i indicate a multicolumn label as y, the model fails.

**Comment by charles9n** [comment]
Currently it expects a single output variable so multiple output variables will fail. Outside of the STS-B task from GLUE, I haven't in general seen a lot of regression tasks for nlp. Is there a public data set with multiple regression labels that you are working with?

**Comment by rnditdev** [comment]
Don't know of any public dataset, the one I'm using is proprietary.

**Comment by rnditdev** [comment]
Maybe get the number of "labels" from the y dataframe, and set it here:
https://github.com/charles9n/bert-sklearn/blob/4a935f28daabf30a1c2a685ed52b3e41c643d925/bert_sklearn/sklearn.py#L191

And then act consequently when building the regressor.

**Comment by charles9n** [comment]
I think its possible for sure. You can try changing it there and see what happens, i.e hardcode it to the dimension of y. You may have to tweak the model loss calc in the model.py file. I would have to try it on some data to see...

**Comment by charles9n** [comment]
Closing due to lack of activity...

**Comment by SaraAmd** [comment]
I have a question on how to use BERT for multi-label regression  Is it possible to do that?
